### PR TITLE
Add failure state

### PR DESCRIPTION
**Pull request by ryelo**
## Prerequisites

- [x] Scope your styles to your individual card
- [x] You are using translations when necessary with proper plural/singular modifications
- [x] You are using the template card provided
- [x] Use functional components & hooks (please)
- [x] Attach screenshots (before and after)
- [x] Make sure that all text in blue is populated with the correct link. If you're unsure what the url should be, just ask!

## What

Adds a state if the API is rejected

### After
Screen Shot 2020-04-16 at 10 32 24 AM

## Additional Info

All of these error states should not actually appear at the same time (unless something horrible happens)

Subscriptions are going to do their own thing because of the opt-in feature

## Code Review
@christiemolloy @ryelo @AllenBW

## Design Review
@kybaker

### CODE REVIEW

File: build/messages/src/Messages.json [hunk 1]
```diff
@@ -1 +1 @@
..."defaultMessage":"stale systems"},{"id":"systemInventoryDanger","description":"System inventory - danger","defaultMessage":"systems to be removed"},{"id":"systemInventoryPercentageData","description":"System Inventory - percentage data","defaultMessage":"0% of total systems"},{"id":"systemInventoryTitle","description":"System Inventory - title","defaultMessage":"Insights system inventory"},{"id":"remediationsTotal","description":"Remediations - remediations total","defaultMessage":"{total} more remediations"},{"id":"remediationsPlaybookStatus","description":"Remediations playbook - status","defaultMessage":"{status}"},{"id":"remediationsPlaybookNoActivity","description":"Remediations playbook - no activity","defaultMessage":"No activity"},{"id":"remediationsNoActivity","description":"Remediations playbook - failed getting data","defaultMessage":"Failure"},{"id":"remediationsNoDataTitle","description":"No remediations","defaultMessage":"You haven't created any remediation Playbooks yet"},{"id":"remediationsNoDataBody","description":"Create a playbook to see remediations","defaultMessage":"Create an Ansible Playbook to remediate or mitigate vulnerabilities or configuration issues."}]
```
Comment (AllenBW) [inline build/messages/src/Messages.json:1]: did that git ignore not... 🤔

File: src/Messages.js [hunk 2]
```diff
@@ -221,5 +221,10 @@ export default defineMessages({
         id: 'remediationsNoDataBody',
         description: 'Create a playbook to see remediations',
         defaultMessage: 'Create an Ansible Playbook to remediate or mitigate vulnerabilities or configuration issues.'
+    },
+    errorStateTitle: {
+        id: 'errorStateTitle',
+        description: 'Error fetching API',
+        defaultMessage: '{appName} has experienced an error. Contact Red Hat support if the problem persists.'
```
Comment (AllenBW) [inline src/Messages.js:228]: 💯

File: src/SmartComponents/Advisor/Advisor.js [hunk 3]
```diff
@@ -48,6 +50,15 @@ const Advisor = ({ recStats, recStatsStatus, advisorFetchStatsRecs, advisorFetch
         }
     }];
 
+    if (advisorIncidentsStatus === 'rejected') {
+        return (
+            <TemplateCard appName='Advisor' data-ouia-safe>
+                <TemplateCardHeader title='Advisor recommendations' />
+                <TemplateCardBody><FailState appName='Advisor'/></TemplateCardBody>
```
Comment (AllenBW) [inline src/SmartComponents/Advisor/Advisor.js:57]: Can we keep keep this on the original return? lil duplication could be eliminated by simply conditionally returning templatecardbody (if status is anything but failed return what exists)

File: src/SmartComponents/PatchManager/PatchManagerCard.js [hunk 4]
```diff
@@ -47,6 +48,15 @@ const PatchManagerCard = ({ systems, systemsStatus, fetchSystems, fetchSecurity,
         chart_color_blue_200.value
     ];
 
+    if (systemsStatus === 'rejected') {
+        return (
```
Comment (AllenBW) [inline src/SmartComponents/PatchManager/PatchManagerCard.js:52]: samesies for this one

File: src/SmartComponents/Vulnerability/VulnerabilityCard.js [hunk 5]
```diff
@@ -64,6 +67,15 @@ const VulnerabilityCard = ({ fetchVulnerabilities, vulnerabilities, vulnerabilit
         }
     }];
 
+    if (vulnerabilitiesFetchStatus === 'rejected') {
+        return (
+            <TemplateCard isFilled={ false } className='ins-c-card__vulnerability' data-ouia-safe>
```
Comment (AllenBW) [inline src/SmartComponents/Vulnerability/VulnerabilityCard.js:72]: samesies for this one

File: src/SmartComponents/Advisor/Advisor.js (hunk 3 above)
Comment (ryelo) [reply src/SmartComponents/Advisor/Advisor.js:57]: we could, right now a few cards are checking the loading state if the status isn't fulfilled, I just didn't rewrite it to do something different

Comment (AllenBW) [reply src/SmartComponents/Advisor/Advisor.js:57]: fair enough, ill followup with some work for advisor, lets get this in

**Pull request merged**